### PR TITLE
add filters for search bar

### PR DESCRIPTION
**Pull request by drewandre**

### CODE REVIEW

File: app/javascript/react/src/containers/SearchBar.js [hunk 1]
```diff
@@ -39,58 +57,78 @@ class SearchBar extends Component {
...
+        .catch(error => this.props.error(error.message));
+        this.props.loading(false);
+      } // end Date.now check
+    } else {
+      this.props.loading(false);
+      this.props.emptyScreen(true);
+      this.setState({ topic: '' })
+      this.setState({ language: '' })
+      this.setState({ onlyReviews: false })
+      this.setState({ disableFields: true })
```
Comment (nwalberts) [inline app/javascript/react/src/containers/SearchBar.js]: WHY. just do it all in one

Comment (drewandre) [reply app/javascript/react/src/containers/SearchBar.js]: @nwalberts Are you referring to lines 73-78 or this entire `handleFormSubmit` function?

Comment (drewandre) [reply app/javascript/react/src/containers/SearchBar.js]: Fixed!

**Comment by chrissyfancy** [comment]
Nice job restructuring this! It looks like you may need some additional tests for your new components however.

**Pull request merged**